service/src/main/java/org/cbioportal/genome_nexus/service/internal/OncokbServiceImpl.java

```diff
@@ -0,0 +1,110 @@
+package org.cbioportal.genome_nexus.service.internal;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.cbioportal.genome_nexus.model.Alteration;
+import org.cbioportal.genome_nexus.persistence.OncokbCancerGenesListRepository;
+import org.cbioportal.genome_nexus.service.OncokbService;
+import org.cbioportal.genome_nexus.service.exception.OncokbNotFoundException;
+import org.cbioportal.genome_nexus.service.exception.OncokbWebServiceException;
+import org.cbioportal.genome_nexus.service.exception.ResourceMappingException;
+import org.cbioportal.genome_nexus.service.remote.OncokbDataFetcher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.ResourceAccessException;
+import org.oncokb.client.CancerGene;
+import org.oncokb.client.IndicatorQueryResp;
+
+@Service
+public class OncokbServiceImpl implements OncokbService {
+    private static final Log LOG = LogFactory.getLog(OncokbServiceImpl.class);
+
+    private final OncokbDataFetcher oncokbDataFetcher;
+    private final OncokbCancerGenesListRepository oncokbCancerGenesListRepository;
+
+    @Autowired
+    public OncokbServiceImpl(OncokbDataFetcher oncokbDataFetcher, OncokbCancerGenesListRepository oncokbCancerGenesListRepository) {
+        this.oncokbDataFetcher = oncokbDataFetcher;
+        this.oncokbCancerGenesListRepository = oncokbCancerGenesListRepository;
+    }
+
+    /**
+     * @param alteration
+     */
```

Comment:
No comment is probably better than an incomplete comment, let's just remove this